### PR TITLE
Save Confluence page's labels in Core

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

See https://github.com/dust-tt/dust/issues/5817.

This PR stores Confluence page labels in core. This is an interim step that allows us to query these labels in the data source block of the Dust app. Currently, we are only taking the first 10 labels, sorted alphabetically. Future enhancements are being tracked [here](https://github.com/dust-tt/tasks/issues/923).

Successfully tested using a data source block.

### CODE REVIEW

File: connectors/src/connectors/confluence/temporal/activities.ts [hunk 1]
```diff
@@ -302,12 +302,25 @@ export async function confluenceCheckAndUpsertPageActivity({
...
+    // Limit to 10 custom tags.
+    const customTags = page.labels.results.slice(0, 10).map((l) => l.id);
+
     const tags = [
       `createdAt:${pageCreatedAt.getTime()}`,
       `space:${spaceName}`,
       `title:${page.title}`,
       `updatedAt:${lastPageVersionCreatedAt.getTime()}`,
       `version:${page.version.number}`,
+      ...customTags,
```
Comment (spolu) [inline connectors/src/connectors/confluence/temporal/activities.ts:323]: Wonder if we want to prefix with something here to avoid any chance of collision?

Comment (flvndvd) [reply connectors/src/connectors/confluence/temporal/activities.ts:323]: Yeah, fair point let me add the `labels:` prefix.

Comment (flvndvd) [reply connectors/src/connectors/confluence/temporal/activities.ts:325]: Done

**Review by spolu — APPROVED** [review]

File: connectors/src/connectors/confluence/temporal/activities.ts [hunk 2]
```diff
@@ -302,12 +302,27 @@ export async function confluenceCheckAndUpsertPageActivity({
       baseUrl: confluenceConfig.url,
       suffix: page._links.tinyui,
     });
+
+    // We log the number of labels to help define the importance of labels in the future.
+    if (page.labels.results.length > 0) {
+      localLogger.info(
+        { labelsCount: page.labels.results.length },
```
Comment (spolu) [inline connectors/src/connectors/confluence/temporal/activities.ts:309]: Maybe add a conditional on whether we will strip or not ?

Comment (flvndvd) [reply connectors/src/connectors/confluence/temporal/activities.ts:309]: I want the count so I can check in DD the P90/P95.

**Pull request merged**